src/app/statistic/sensor-statistic/sensor-statistic.template.html

```diff
@@ -1,8 +1,8 @@
 <div class="chart-holder">
-    <sh-servo-statistic-column-chart *ngIf="sensorId === 'servo'"
+    <sh-switcher-statistic-column-chart *ngIf="sensorId === 'switcher'"
```

Comment:
move this if expression in the component controller under the function/property named `isSwitcherWidgetVisible`